### DISCUSSION TITLE
Release 3.5.0-beta - catalogue search needs to be opened twice

### DISCUSSION THREAD

**Discussion by aetherfaerber** [kitodo/kitodo-production#5463] [General]
After switching to the 3.5.0-beta release we are experiencing the following oddity:

1. Open a new process/„Vorgang“.
2. Click the data transfer („Datenübernahme“) button, open catalogue search.
3. The pop-up menu allows entries in the search field etc. but the Search OPAC (maybe this label should be changed?) button stays greyed out and cannot be clicked even with all values set/changed.
4. Choose Cancel.
5. Open catalogue search again.
6. Values are still set and button is now active and can be clicked.

We have set up an import configuration with a search interface of type CUSTOM and metadata format OTHER. We did not use the configuration importer to source our old kitodo_opac.xml but created a fresh configuration through the webinterface.

So far I have no idea if this i a bug or caused by a configuration on our side.

**Comment by solth** [comment]
@aetherfaerber thanks for reporting this. I could reproduce this behaviour, so I will create an issue from this discussion.

**Comment by solth** [comment]
Documented in #5485 and fixed in release 3.5.0 via #5486
